x-pack/plugins/observability/public/components/threshold/threshold_rule_expression.tsx

```diff
@@ -399,13 +418,32 @@ export default function Expressions(props: Props) {
         </h5>
       </EuiTitle>
       <EuiSpacer size="s" />
-      <RuleFlyoutKueryBar
+      <SearchBar
+        appName={i18n.translate(
+          'xpack.observability.threshold.rule.alertFlyout.searchBar.appName',
+          {
+            defaultMessage: 'Threshold rule',
```

Comment:
This seems to be the app name, perhaps it should be `Observability`?

Reply:
I don't think we need to have the same app name as it is used, but rather, it is an id for the search bar, so Threshold rule should be fine in this case.

Reply:
It seems to be used to [report UI events](https://github.com/elastic/kibana/blob/0c2d483d19528eb970ba73dca4462a2eb1865239/src/plugins/usage_collection/public/plugin.tsx#L91), so `Threshold rule` should be fine. We can change it to `Custom Threshold Rule` perhaps.